.github/workflows/main.yaml

```diff
@@ -7,11 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up JDK 8
+      - name: Set up JDK 11
         uses: actions/setup-java@v3
         with:
-          java-version: '8'
-          distribution: 'temurin'
+          java-version: '11'
```

Comment:
- The sdk i rely on to connect to Astra uses java11+. I make sure to only use java11 in my module by overriding the compiler plugin but it impacts mostly the build - other stores can in Java8

But overall i think it would be a good choice to target new versions of java: 

- Java 11 will go EOL in 2024. It is really time to move on. Better would be to go directly to go Java17 and look for support of GraalVM. 
- We are racing we the python world and need to be as performant as possible.
- Some in my company create [JLAMA](https://github.com/tjake/Jlama) and went directly with Java20 to benefit from Panama threads.